### ISSUE TITLE
About the verification set

### ISSUE THREAD

**Issue by fafu77** [vvgoder/SEU_PML_Dataset#2]
Hi, author. We saw in your paper that you divided about 4610 images into the training set and 659 images into the validation set. However, we can only find the download link of the train set (all license plates are mosaicked off) on github, which has a total of 5269 pictures, and does not divide the training set and the verification set. This leaves us really confused, can you provide your divided training set and validation set? Only in this way can the next comparative experiment be carried out reasonably. Thanks.

**Comment by vvgoder** [comment]
> Hi, author. We saw in your paper that you divided about 4610 images into the training set and 659 images into the validation set. However, we can only find the download link of the train set (all license plates are mosaicked off) on github, which has a total of 5269 pictures, and does not divide the training set and the verification set. This leaves us really confused, can you provide your divided training set and validation set? Only in this way can the next comparative experiment be carried out reasonably. Thanks.

Thank you for you attention to our work, I've uploaded two txt files that embrace the image name of training set and validation set.
